template/package.json

```diff
@@ -14,7 +14,7 @@
     {{#unit}}
     "test": "karma start karma.conf.js",
     {{/unit}}
-    "build": "cross-env NODE_ENV=production browserify -g envify -p [ vueify/plugins/extract-css -o dist/build.css ] -e src/main.js | uglifyjs -c warnings=false -m > dist/build.js"
+    "build": "cross-env NODE_ENV=production browserify -g envify -t [ babelify ] -t [ vueify ] -p [ vueify/plugins/extract-css -o dist/build.css ] -e src/main.js | uglifyjs -c warnings=false -m > dist/build.js"
   },
   "browserify": {
     "transform": [
```

Comment:
@ivangabriele these two are already specified here under `transform`. How come you needed to add it into `build` script manually?